Resources/Prototypes/Entities/Structures/Machines/vending_machines.yml

```diff
@@ -200,6 +202,10 @@
   components:
   - type: VendingMachine
     pack: ClothesMateInventory
+    offState: off
+    brokenState: broken
+    normalState: normal-unshaded
```

Comment:
these 3 seem to be basically the same for all of these, can it not just default?

Reply:
yeah i suppose this is fine